src/components/Support/Popover/index.tsx

```diff
@@ -20,6 +24,7 @@ const Popover: React.FC<IPopoverProps> = ({ children, ...restProps }) => {
   const close = (): void => setOpened(false)
 
   return (
+    // @ts-expect-error Error is caused by package and out of our control
```

Comment:
I couldn't figure out how to clear this error because it seemed to be caused by the fact that `ReactPopover` doesn't expect children. But the package here isn't very well maintained... The only thing I could come up with was ignoring the error but does anyone have a better alternative? 

```
No overload matches this call.
  Overload 1 of 2, '(props: PopoverProps | Readonly<PopoverProps>): Popover', gave the following error.
    Type '{ children: Element; body: ReactNode; enterExitTransitionDurationMs: number; isOpen: boolean; onOuterAction: () => void; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Popover> & Readonly<PopoverProps>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Popover> & Readonly<PopoverProps>'.
  Overload 2 of 2, '(props: PopoverProps, context: any): Popover', gave the following error.
    Type '{ children: Element; body: ReactNode; enterExitTransitionDurationMs: number; isOpen: boolean; onOuterAction: () => void; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Popover> & Readonly<PopoverProps>'.
      Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Popover> & Readonly<PopoverProps>'.
```